libnetwork/datastore/datastore.go

```diff
@@ -261,6 +261,29 @@ func (ds *Store) Map(key string, kvObject KVObject) (map[string]KVObject, error)
 	return results, nil
 }
 
+// DeleteObject deletes a kvObject from the on-disk DB and the in-memory cache.
+// Unlike DeleteObjectAtomic, it doesn't check the optimistic lock of the
+// passed kvObject.
+func (ds *Store) DeleteObject(kvObject KVObject) error {
+	ds.mu.Lock()
+	defer ds.mu.Unlock()
+
+	if kvObject == nil {
+		return types.InvalidParameterErrorf("invalid KV Object : nil")
```

Comment:
Looks like a stray space here,. or was this intentional?

```suggestion
		return types.InvalidParameterErrorf("invalid KV Object: nil")
```

Reply:
Ah, I see it's probably to match the existing errors, which also have this 🙈 

Yeah, we can probably fix all of them in a follow-up (and could consider using `errdefs` directly instead of the `types` package)